### ISSUE TITLE
DetrImageProcessor.normalize_annotation() got an unexpected keyword argument 'input_data_format'

### ISSUE THREAD

**Issue by r-remus** [huggingface/transformers#25731]
### System Info
- `transformers` version: 4.32.0
- Platform: Linux-5.15.0-79-generic-x86_64-with-glibc2.35
- Python version: 3.10.12
- Huggingface_hub version: 0.16.4
- Safetensors version: 0.3.3
- Accelerate version: not installed
- Accelerate config: not found
- PyTorch version (GPU?): 2.0.1+cu117 (True)
- Tensorflow version (GPU?): not installed (NA)
- Flax version (CPU?/GPU?/TPU?): not installed (NA)
- Jax version: not installed
- JaxLib version: not installed
- Using GPU in script?: <fill in>
- Using distributed or parallel set-up in script?: <fill in>

### Who can help?
@amyeroberts

### Information

- [ ] The official example scripts
- [X] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [X] My own task or dataset (give details below)

### Reproduction

`DetrImageProcessor.normalize_annotation()` doesn't take the new `input_data_format` which is passed to it.

Code to reproduce the issue:
```python
from transformers import DetrImageProcessor
image_processor = DetrImageProcessor()
_ = image_processor(images=[img], annotations=[{'image_id': 0, 'annotations': []}], return_tensors="pt")
```

### Expected behavior

It should be possible to pass `input_data_format` to `DetrImageProcessor.normalize_annotation()`.

**Comment by amyeroberts** [comment]
@r-remus The fix has now been included as part of a [patch release](https://github.com/huggingface/transformers/releases/tag/v4.32.1) and can be directly installed from pypi with `pip install transformers`

**Comment by r-remus** [comment]
Sorry for getting back to you so late.
@ArthurZucker: No, I wasn't using main but release v4.32.0.
@amyeroberts: Thanks, that's great! Works for me now (using v4.33.2 now, tho).
